src/screens/game-details-modal/index.tsx

```diff
@@ -182,6 +184,41 @@ function GameDetailsStreamButton({ match }: { match: CoreData.Match }) {
   );
 }
 
+function GameDetailsReplayButton({ match }: { match: CoreData.Match }) {
+  const styles = useStyles(getStyles);
+  const translate = useTranslate();
+
+  const { replayVideo, openReplayVideo, searchReplay } = useReplay();
+  useEffect(() => {
+    searchReplay({
+      date: new Date(match.date),
+      teams: match.teams,
+      game: match.matchDetails.competitionName,
+    });
+  }, [replayVideo, openReplayVideo, searchReplay]);
```

Comment:
You don't need to add `replayVideo` and `openReplayVideo` to dependencies (it could reduce performance). You need to add `match` to dependencies.